qui/tray/devices.py

```diff
@@ -123,45 +122,49 @@ def dev_detached(self, vm):
 
     def toggle(self, menu_item):
         if menu_item.attached:
-            self.detach()
+            self.detach_item()
         else:
-            self.attach(menu_item)
+            self.attach_item(menu_item)
 
-    def attach(self, menu_item):
-        self.detach()
+    def attach_item(self, menu_item):
+        self.detach_item()
 
         try:
             assignment = qubesadmin.devices.DeviceAssignment(
                 self.device.backend_domain, self.device.ident, persistent=False)
             menu_item.vm.devices[menu_item.devclass].attach(assignment)
             subprocess.call(
                 ['notify-send',
-                 "Attaching %s to %s" % (self.device.description, menu_item.vm)])
+                 "Attaching %s to %s" % (
+                     self.device.description, menu_item.vm)])
         except exc.QubesException as ex:
             subprocess.call(
                 ['notify-send', '-t', '15000', '-i', 'dialog-error',
                  'Attaching device {0} to {1} failed. Error: {2}'.format(
                      self.device.description,
                      menu_item.vm,
                      ex)])
-        except Exception as ex:
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
You don't use `ex` here, so you can use just `except Exception:`. I wonder why pylint doesn't complain about this here.